docs/server/source/simple-network-setup.md

```diff
@@ -240,6 +244,45 @@ Note: We'll share some example startup scripts in the future. This document is a
 
 If you followed the recommended approach and created startup scripts for BigchainDB and Tendermint, then you can reboot the machine now. MongoDB, BigchainDB and Tendermint should all start.
 
+
+### Member: Start BigchainDB and Tendermint using Monit
+
+This section describes how to manage the BigchainDB and Tendermint processes using [Monit][monit] - a small open-source utility for managing and monitoring Unix processes.
+
+This section assumes that you followed the guide down to the [start MongoDB section](member-start-mongodb) inclusive.
+
+Install Monit:
+
+```
+sudo apt install monit
+```
+
+If you installed the `bigchaindb` Python package, you should have the `bigchaindb-monit-config` script in your `PATH` now.
+
+Run the script:
+
+```
+bigchaindb-monit-config
+```
+
+The script builds a configuration file for Monit.
+
+Run Monit as a daemon, instructing it to wake up every second to check on processes:
+
+```
+monit -d 1
+```
+
+It will run the processes and restart them when they crash. If the root `bigchaindb_` process crashes, Monit will also restart the Tendermint process.
+
+Check the status by running `monit status` or `monit summary`.
+
+By default, it will collect program logs into the `~/.bigchaindb-monit/logs` folder.
```

Comment:
I looked in some of those logs and was surprised to find that some of the "main" BigchainDB logs were not in `bigchaindb.out.log` but in `bigchaind-benchmark.log`, e.g.

```bash
~/.bigchaindb-monit/logs$ cat bigchaindb.out.log
======== Running on http://localhost:9985 ========
(Press CTRL+C to quit)

~/.bigchaindb-monit/logs$ cat bigchaindb-benchmark.log 
2018-07-27 14:40:03, INFO, BigchainDB Version 2.0.0b3
2018-07-27 14:40:03, INFO, Initializing database
2018-07-27 14:40:03, INFO, Create database `bigchain`.
2018-07-27 14:40:03, INFO, Create `transactions` table.
2018-07-27 14:40:03, INFO, Create `utxos` table.
2018-07-27 14:40:03, INFO, Create `assets` table.
2018-07-27 14:40:03, INFO, Create `blocks` table.
2018-07-27 14:40:03, INFO, Create `metadata` table.
2018-07-27 14:40:03, INFO, Create `validators` table.
2018-07-27 14:40:03, INFO, Create `pre_commit` table.
2018-07-27 14:40:03, INFO, Create `transactions` secondary index.
2018-07-27 14:40:03, INFO, Create `assets` secondary index.
2018-07-27 14:40:03, INFO, Create `assets` secondary index.
2018-07-27 14:40:03, INFO, Create `utxos` secondary index.
2018-07-27 14:40:03, INFO, Create `pre_commit` secondary index.
2018-07-27 14:40:04, INFO, Create `validators` secondary index.
2018-07-27 14:40:04, INFO, Starting BigchainDB main process.
2018-07-27 14:40:04, INFO, 
****************************************************************************
*                                                                          *
*   ┏┓ ╻┏━╸┏━╸╻ ╻┏━┓╻┏┓╻╺┳┓┏┓    ┏━┓ ┏━┓ ╺┳┓┏━╸╻ ╻                         *
*   ┣┻┓┃┃╺┓┃  ┣━┫┣━┫┃┃┗┫ ┃┃┣┻┓   ┏━┛ ┃┃┃  ┃┃┣╸ ┃┏┛                         *
*   ┗━┛╹┗━┛┗━╸╹ ╹╹ ╹╹╹ ╹╺┻┛┗━┛   ┗━╸╹┗━┛╹╺┻┛┗━╸┗┛                          *
*   codename "fluffy cat"                                                  *
*   Initialization complete. BigchainDB Server is ready and waiting.       *
*                                                                          *
*   You can send HTTP requests via the HTTP API documented in the          *
*   BigchainDB Server docs at:                                             *
*    https://bigchaindb.com/http-api                                       *
*                                                                          *
*   Listening to client connections on: 0.0.0.0:9984                       *
*                                                                          *
****************************************************************************

2018-07-27 14:40:04, INFO,  ABCIServer started on port: 26658
2018-07-27 14:40:04, WARNING, WebSocket connection failed with exception Cannot connect to host localhost:26657 ssl:False [Connect call failed ('127.0.0.1', 26657)]
2018-07-27 14:40:06, INFO,  ... connection from Tendermint: 127.0.0.1:60950 ...
2018-07-27 14:40:06, INFO,  ... connection from Tendermint: 127.0.0.1:60952 ...
2018-07-27 14:40:06, INFO,  ... connection from Tendermint: 127.0.0.1:60954 ...
2018-07-27 14:40:07, INFO, Connected to tendermint ws server
```

Reply:
We did not change anything in this area. We do need to tidy the logging up a little in the future.

Reply:
I will I create a new issue that links to this review comment and then we can consider this comment taken-care-of for this PR.

Reply:
See https://github.com/bigchaindb/bigchaindb/issues/2417